### PR TITLE
Swope build new content, accuracy changes

### PR DESCRIPTION
**Pull request by swopebe**

### CODE REVIEW

File: applications/ansible_config.adoc [hunk 1]
```diff
@@ -115,16 +115,14 @@ spec:
...
+To launch an {aap-short} Workflow by using the `AnsibleJob` custom resource, replace the `job_template_name` field with the `workflow_template_name`, which is displayed in the following example.
 
 [#workflow-sample-yaml]
 == Using {aap-short} sample YAML Workflow
 
-See the following sample of a Workflow `AnsibleJob` `.yaml` file in a Git prehook and Git posthook folder:
+See the following sample of a Workflow `AnsibleJob` YAML file in a Git prehook and Git posthook folder:
 
 [source,yaml]
 ----
```
Comment (swopebe) [inline applications/ansible_config.adoc:128]: @mikeshng is the API still "tower"?

File: applications/ansible_setup.adoc [hunk 2]
```diff
@@ -11,27 +11,25 @@ With {aap-short} jobs, you can automate tasks and integrate with external servic
...
 [#install-ansible]
 == Installing {aap-short} Resource Operator
 
 . Log in to your {ocp-short} cluster console.
 . Click *OperatorHub* in the console navigation.
-. Search for and install the _{aap-short} Resource Operator_. *Note:* To submit prehook and posthook `AnsibleJobs`, install {aap} Resource Operator with corresponding version available on different {ocp-short} versions: 
-//these likely change, too:
+. Search for and install the _{aap-short} Resource Operator_. *Note:* To submit prehook and posthook `AnsibleJobs`, install {aap} Resource Operator with corresponding version available on the following {ocp-short} versions: 
 
   - {ocp-short} 4.6 needs (AAP) Resource Operator early-access
```
Comment (swopebe) [inline applications/ansible_setup.adoc]: @mikeshng I think we can remove 4.6 and 4.7 here? (line 30 and after)

File: applications/ansible_setup.adoc [hunk 3]
```diff
@@ -11,27 +11,25 @@ With {aap-short} jobs, you can automate tasks and integrate with external servic
...
 
   - {ocp-short} 4.6 needs (AAP) Resource Operator early-access 
   - {ocp-short} 4.7 needs (AAP) Resource Operator early-access, stable-2.1
   - {ocp-short} 4.8 needs (AAP) Resource Operator early-access, stable-2.1, stable-2.2
   - {ocp-short} 4.9 needs (AAP) Resource Operator early-access, stable-2.1, stable-2.2 
   - {ocp-short} 4.10 needs (AAP) Resource Operator stable-2.1, stable-2.2
 
-You can then create the credential from the _Credentials_ page in the console. Click *Add credential* or access the page from the navigation. See link:../multicluster_engine/credentials/credential_ansible.adoc[Creating a credential for {aap-short} ] for credential information.
\ No newline at end of file
+You can then create the credential from the _Credentials_ page in the console. Click *Add credential*, or access the page from the navigation. See link:../multicluster_engine/credentials/credential_ansible.adoc[Creating a credential for {aap-short}] for credential information.
```
Comment (swopebe) [inline applications/ansible_setup.adoc:33]: It's odd to be linking from app to MCE but this link will likely change with MCE changes...PM wants this to be more link "clusters". I will revisit with Josh on situations like this.

File: applications/ansible_config.adoc (hunk 1 above)
Comment (mikeshng) [reply applications/ansible_config.adoc:128]: yes the API still has the tower reference that did not change.

File: applications/ansible_setup.adoc (hunk 2 above)
Comment (mikeshng) [reply applications/ansible_setup.adoc]: If that's no longer supported. Yes, please remove them. Thanks.

File: applications/ansible_setup.adoc [hunk 4]
```diff
@@ -11,27 +11,25 @@ With {aap-short} jobs, you can automate tasks and integrate with external servic
 [#prerequisites-for-ansible-integration]
```
Comment (oafischer) [inline applications/ansible_setup.adoc:11]: We can probably shorten this anchor.

**Review by oafischer — APPROVED** [review]
lgtm

**Pull request merged**